eng/common/scripts/add-pullrequest-reviewers.ps1

```diff
@@ -1,61 +0,0 @@
-param(
```

Comment:
Lets make sure there are no other users of this script that we will break by removing this.

Reply:
Its only ever used in the `create-pull-request.yml` file.